Cython/Compiler/MemoryView.py

```diff
@@ -756,7 +756,7 @@ def _resolve_NameNode(env, node):
     except AttributeError:
         raise CompileError(node.pos, INVALID_ERR)
 
-    viewscope = env.global_scope().context.cython_scope.viewscope
+    viewscope = env.context.cython_scope.viewscope
```

Comment:
This is unrelated to this PR, but – do we really need to go through the context for this? Is there no direct way just through the scopes? :-o